scss/splitter/_layout.scss

```diff
@@ -131,4 +132,45 @@
         opacity: 0;
         position: absolute;
     }
+
+    // flexbox-based splitter
+
+    .k-splitter-flex {
+        display: flex;
+        width: 100%;
+        height: auto;
+
+        .k-pane {
+            flex: 1 1 auto;
+            min-width: 0;
+            max-width: 100%;
+            min-height: 0;
+            max-height: 100%;
+        }
+
+        .k-pane-static {
+            flex-grow: 0;
+            flex-shrink: 0;
+        }
+
+        .k-splitbar {
+            position: static;
+            flex: 0 0 auto;
+        }
+
+        &.k-splitter-horizontal {
+            flex-direction: row;
+
+            &[dir=rtl] {
+                flex-direction: row-reverse;
+            }
+        }
+        &.k-splitter-vertical {
+            flex-direction: column;
+
+            &[dir=rtl] {
+                flex-direction: column-reverse;
```

Comment:
That should be removed. The rest is fine

Reply:
Very nice catch, thanks!